django/db/backends/mysql/schema.py

```diff
@@ -34,7 +34,7 @@ def quote_value(self, value):
         # MySQLdb escapes to string, PyMySQL to bytes.
         quoted = self.connection.connection.escape(value, self.connection.connection.encoders)
         if isinstance(value, str) and isinstance(quoted, bytes):
-            quoted = quoted.decode()
+            quoted = quoted.decode().replace('%', '%%')
```

Comment:
Do we need this change?

Reply:
Yes, otherwise this test will break:
`migrations.test_operations.OperationTests.test_add_constraint_percent_escaping`